synapse/server.py

```diff
@@ -140,7 +140,8 @@ def cache_in_self(builder: T) -> T:
             "@cache_in_self can only be used on functions starting with `get_`"
         )
 
-    depname = builder.__name__[len("get_") :]
+    # get_attr -> _attr
```

Comment:
Could change that to this for better clarification:
```suggestion
    # get_variable() -> _variable
```